### PR TITLE
Rewritten README.md to be more informative and useful

### PR DESCRIPTION
**Pull request by AlexC**
I've done this in ref to bug #4486, which provides a much more informative and useful README.md file (one that is also up-to-date).

This could also be merged into 3.3/develop

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
-# Kohana PHP Framework, version 3.2 (development)
+# Kohana PHP Framework
 
-This is the current development version of [Kohana](http://kohanaframework.org/).
+[Kohana](http://kohanaframework.org/) is an elegant, open source and object oriented HMVC framework built using PHP5, by a team of volunteers that aims to be swift, secure and small.
```
Comment (shadowhand) [inline README.md]: Please use Oxford commas:

> ... an elegant, open source_,_ and object oriented ...

File: README.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
 
-For the most current release, see the 3.1/master branch.
+Released under a [BSD license](http://kohanaframework.org/license), Kohana can be used legally for any open source, commercial or personal project.
```
Comment (shadowhand) [inline README.md]: Also Oxford comma after the "or".

File: README.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
-# Kohana PHP Framework, version 3.2 (development)
+# Kohana PHP Framework
 
-This is the current development version of [Kohana](http://kohanaframework.org/).
+[Kohana](http://kohanaframework.org/) is an elegant, open source, and object oriented HMVC framework built using PHP5, by a team of volunteers that aims to be swift, secure and small.
 
-For the most current release, see the 3.1/master branch.
+Released under a [BSD license](http://kohanaframework.org/license), Kohana can be used legally for any open source, commercial or, personal project.
```
Comment (shadowhand) [inline README.md]: Comma goes before "or", not after.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,19 @@
-# Kohana PHP Framework, version 3.2 (development)
+# Kohana PHP Framework
 
-This is the current development version of [Kohana](http://kohanaframework.org/).
+[Kohana](http://kohanaframework.org/) is an elegant, open source, and object oriented HMVC framework built using PHP5, by a team of volunteers that aims to be swift, secure and small.
```
Comment (shadowhand) [inline README.md]: Also Oxford comma and the end of this line:

> swift, secure, and small

**Comment by AlexC** [comment]
Ok, those 2 last changes are done ... I did however force push them to my branch.

File: README.md [hunk 5]
```diff
@@ -1,5 +1,19 @@
-# Kohana PHP Framework, version 3.2 (development)
+# Kohana PHP Framework
 
-This is the current development version of [Kohana](http://kohanaframework.org/).
+[Kohana](http://kohanaframework.org/) is an elegant, open source, and object oriented HMVC framework built using PHP5, by a team of volunteers that aims to be swift, secure, and small.
```
Comment (shadowhand) [inline README.md]: Thanks for letting me be a grammar Nazi... this line still doesn't read well:

> HMVC framework built using PHP5, by a team of volunteers that aims to be swift, secure, and small.

This implies that the "team of volunteers" aims to be switch, secure, and small. Lets rewrite it as:

> PHP HMVC framework built by a team of volunteers. It aims to be swift, secure, and small.

File: README.md [hunk 6]
```diff
@@ -1,5 +1,19 @@
...
 
-For the most current release, see the 3.1/master branch.
+Released under a [BSD license](http://kohanaframework.org/license), Kohana can be used legally for any open source, commercial, or personal project.
+
+## Documentation
+Kohana's documentation can be found at http://kohanaframework.org/documentation which also contains an API browser.
```
Comment (shadowhand) [inline README.md]: Links should be wrapped with brackets:

```
<http://kohanframework.org/documentation>
```

Comment (AlexC) [inline README.md]: I'm a little confused as to why - can you explain?

Comment (shadowhand) [inline README.md]: Because it is proper Markdown syntax. Not all Markdown implementations will convert raw links into anchors without the brackets.

**Comment by AlexC** [comment]
Ok, there we go - changes done.

**Comment by shadowhand** [comment]
Thanks!

**Comment by shadowhand** [comment]
Merged up to 3.3/develop in f538835ad3fa64a9dc71b31f31eb012afb9830d8

**Comment by zeelot** [comment]
You forgot the fact that "log in" is two words :(

**Pull request merged**